scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -512,6 +512,70 @@ task ValidateFilterSetName {
     }
 }
 
+task IsVQSRLite {
+  input {
+    String project_id
+    String fq_filter_set_info_table
+    String filter_set_name
+  }
+  meta {
+    # Not `volatile: true` since there shouldn't be a need to re-run this if there has already been a successful execution.
+  }
+
+  # add labels for DSP Cloud Cost Control Labeling and Reporting
+  String bq_labels = "--label service:gvs --label team:variants --label managedby:gvs_utils"
+  File monitoring_script = "gs://gvs_quickstart_storage/cromwell_monitoring_script.sh"
```

Comment:
nit: do you really need the monitoring script for this task?

Reply:
No, not really!